leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/CaliforniumEndpointsManager.java

```diff
@@ -137,7 +139,66 @@ public boolean isTrusted(RawPublicKeyIdentity id) {
                 newBuilder.setIdentity(serverInfo.privateKey, new Certificate[] { serverInfo.clientCertificate });
 
                 // set X509 verifier
-                newBuilder.setCertificateVerifier(new DefaultLeshanCertificateVerifier(serverInfo.serverCertificate));
+
+                // LWM2M v1.1.1 - 5.2.8.7. Certificate Usage Field
+                //
+                // 0: Certificate usage 0 ("CA constraint")
```

Comment:
would this need the pre tags too?

Reply:
It seems this is not needed. At least with eclipse, I mean eclipse IDE don't touch this kind of none javadaoc comment.